setup.py

```diff
@@ -102,6 +115,16 @@
 elif platform.system() == 'Windows':
     ssw_extra_compile_args.extend(['-openmp:experimental'])
 
+stats_extra_compile_args = []+ssw_extra_compile_args
+stats_extra_link_args = []
+if platform.system() != 'Windows':
+    if icc or sysconfig.get_config_vars()['CC'] == 'icc':
+        stats_extra_compile_args.extend(['-qopenmp'])
+        stats_extra_link_args.extend(['-qopenmp'])
+    elif not clang:
+        stats_extra_compile_args.extend(['-fopenmp'])
+        stats_extra_link_args.extend(['-fopenmp'])
```

Comment:
```suggestion
    else:
        stats_extra_compile_args.extend(['-fopenmp-simd'])
        stats_extra_link_args.extend(['-fopenmp-simd'])
```
(supported by both GCC 4.9+ and clang 6+)

Reply:
Nope, we need proper openmp, not just sims.